pkg/client/kubernetes/client.go

```diff
@@ -174,7 +186,7 @@ func RESTConfigFromKubeconfig(kubeconfig []byte) (*rest.Config, error) {
 		return nil, err
 	}
 
-	if err := validateClientConfig(clientConfig); err != nil {
+	if err := validateClientConfig(clientConfig, []string{}); err != nil {
```

Comment:
Nit: Instead of passing an empty slice maybe we can use pass `nil`?

(same for others)